src/lib/ketrew_configuration.mli

```diff
@@ -180,3 +142,47 @@ val connection: client -> string
 val token: client -> string
 
 val standalone_of_server: server -> standalone
+
+
+val load_exn:
+  ?and_apply:bool ->
+  ?profile:string ->
+  [ `From_path of string
+  | `Guess
+  | `In_directory of string
+  | `Override of t ] ->
+  t
+(** Load a configuration.
```

Comment:
3rd point. If the callers have a configuration don't load it. Whatever __side effect__ is being performed by _loading_ should be exposed as a separate function, or refactored entirely.